### DISCUSSION TITLE
Idea to add: Hide left "tree bar", and "notes bar" by keybord-shortcut

### DISCUSSION THREAD

**Discussion by mrutkowsk** [massCodeIO/massCode#349] [Ideas]
**First thank You for Your awesome program.**
Reasons. When I do research, often I split screen and left part is open Firefox with article, and right part is open **masscode**.
Problem I have is _"treebar"_ and notes bar are taking significant portion of the designated area and would be super beneficial to have a feature to hid them by shortcut or some simple   button accessible from everywhere .

**Goal:**
when working on single note, to  be able to see just a note. real-estate on small monitors / laptops is important, and this would be very beneficial to everybody who is using Your software on regular basis like me.

**Regards :)**
![no tree bar ides](https://user-images.githubusercontent.com/66516742/222845941-8cf5f59d-8f6b-4857-83d8-627662601283.png)

**Comment by rtpHarry** [comment]
I second this - I have to use a separate app to take notes during meetings, because if im screen sharing i expose all my categories and note headings to other clients when i need to add a note. I want a sidebar free mode to be able to use this openly in meetings.

**Comment by antonreshetov** [comment]
Implemented #610
